frappe/patches/v11_0/apply_customization_to_custom_doctype.py

```diff
@@ -30,7 +30,7 @@ def execute():
 			property_setter_map[prop.field_name] = prop
 			frappe.db.sql('DELETE FROM `tabProperty Setter` WHERE `name`=%s', prop.name)
 
-		meta = frappe.get_doc('DocType', doctype.name)
+		meta = frappe.get_meta(doctype.name)
```

Comment:
should not be meta because meta is loaded with `Property Setter` applied

Reply:
already part of upstream/master

https://github.com/frappe/frappe/blob/master/frappe/patches/v11_0/apply_customization_to_custom_doctype.py#L33

fix for cloud users by @surajshetty3416